labgrid/driver/flashromdriver.py

```diff
@@ -0,0 +1,60 @@
+# pylint: disable=no-member
+import subprocess
+import os.path
+import attr
+import logging
+
+from ..resource import Flashrom, NetworkFlashrom
+from ..factory import target_factory
+from ..step import step
+from .common import Driver, check_file
+from ..util.managedfile import ManagedFile
+
+
+@target_factory.reg_driver
+@attr.s(cmp=False)
+class FlashromDriver(Driver):
```

Comment:
Drivers should implement a Protocol. For this driver, BootstrapProtocol would be the most fitting, however it would require a rename of `flash` to `load`. I am also open to a new protocol suggestion, since this can not only be used to bootstrap the device.